sktime/utils/model_lists.py

```diff
@@ -0,0 +1,40 @@
+from sktime.classification import (OSCNNClassifier,
+                                   OSCNNRegressor)
+def construct_all_classifiers(nb_epochs=None):
```

Comment:
I do not think we need this file in `sktime`, we have the tests `TestAllClassifiers` etc.

Reply:
ok, I'll add the file contents to `TestAllClassifiers` as required then?